_posts/2020-02-14-aviatar-experiences-significant-savings.adoc

```diff
@@ -0,0 +1,70 @@
+
+== Why not Spring Boot?
+
+Lufthansa Technik see Quarkus as a great fit in their journey to running their services in a serverless mode. They have many services that aren’t invoked that often but they still need to have 3 instances of each up and running all the time for high-availability requirements leading to high cloud resource consumption costs. They plan to turn these seldom accessed services to Function-as-a-service calls so that they can be invoked on demand leading to a reduction of cloud. If they were using Spring Boot, it would take too long to boot up, making it prohibitive to use in a serverless mode.
+ 
+Likewise, they have experienced lower memory and compute cloud resource consumption when using Quarkus and according to Thorsten, _“with Quarkus, they could run 3 times denser deployments without sacrificing availability and response times of services.”_
+ 
+Their developers are Spring Boot developers with Java EE experience as well, so the learning curve for Quarkus was very small since its syntax and approach was _“close to what our developers are already doing and it’s familiar to them. This is a big benefit,”_ Thorsten affirmed.
+ 
+With respect to the recently introduced Spring API compatibility feature in Quarkus, Thorsten said that _“they may use the Spring API compatibility in Quarkus when migration their current Spring Boot microservices to Quarkus. However, for developing new microservices, they plan to use just the Quarkus APIs directly because it would be awkward to use another API within Quarkus.”_
+
+
+== The Quarkus Benefits
+
+Among the many benefits that Quarkus provides to Lufthansa Technik, Thorsten mentioned that _“they could cut cloud resource costs threefold."_ And the same goes with OpenShift because of the higher density you can achieve on each core using Quarkus. A tangible example is for the one mentioned earlier for an existing microservice that was consuming ½ core plus 1 GB of RAM per instance, which required 1.5 cores and 3 GB of RAM when running it in HA on the cloud (3 instances). When using the Quarkus version of the same microservice, its consumption was of 200 milicores plus 200-400 MB of RAM per instance. This translates to 0.6 cores plus 600 MB – 1.2 GB of RAM for an HA deployment of 3 instanced of the microservice. _“They could run 3 times denser deployments without sacrificing availability and response times of services,”_ Thornsten reiterated.
```

Comment:
```suggestion
Among the many benefits that Quarkus provides to Lufthansa Technik, Thorsten mentioned that _“they could cut cloud resource costs threefold."_ And the same goes with OpenShift because of the higher density you can achieve on each core using Quarkus. A tangible example is for the one mentioned earlier for an existing microservice that was consuming ½ core plus 1 GB of RAM per instance, which required 1.5 cores and 3 GB of RAM when running it in HA on the cloud (3 instances). When using the Quarkus version of the same microservice, its consumption was of 200 millicores plus 200-400 MB of RAM per instance. This translates to 0.6 cores plus 600 MB – 1.2 GB of RAM for an HA deployment of 3 instances of the microservice _“. They could run 3 times denser deployments without sacrificing availability and response times of services”_, Thornsten reiterated.
```